### DISCUSSION TITLE
Releases are not showing recent releases or past releases

### DISCUSSION THREAD

**Discussion by ericoporto** [emscripten-core/emscripten#16001] [Q&A] [answered]
I remember the Releases page having all releases, not sure if I am misremembering. I checked the tags, and there are newer releases (there are older releases too). I checked [`docs/process.md`](https://github.com/emscripten-core/emscripten/blob/main/docs/process.md) but it doesn't mention GitHub releases.

Is there been any workflow changes? Should I use the numbers on tags as releases, rely on information on the change notes [here](https://emscripten.org/docs/introducing_emscripten/release_notes.html)?

**Comment by sbc100** [comment, marked as answer]
Yes, we don't make release, only tags.    If you are trying to use emscripten directly from git or package emscripten then that tags are the way to go.  They correspond to the prebuilt versions that are available in emsdk.
